policyengine_us/parameters/gov/states/hi/tax/income/credits/food_excise_tax/minor_child/amount.yaml

```diff
@@ -0,0 +1,14 @@
+description: Hawaii provides this amount under the food excise tax credit for each minor child receiving public support.
+values:
+  2022-01-01: 110
+metadata:
+  unit: currency-USD
+  label: Hawaii excise tax credit minor child amount
+  reference:
+    - title: Hawaii Income Tax Law, Chapter 235, Section 235-55.85, (b)
+      href: https://files.hawaii.gov/tax/legal/hrs/hrs_235.pdf#page=44
```

Comment:
Subsection (b) doesn't mention $110. I think this is more in subsection (c), and then the department of taxation interpreted it as those children being effectively exemptions with zero income, assigning them to the full value for the tax form. Leave a comment explaining this.

Reply:
I agreed. It should be subsection (c). Thank you!